helm/aws-load-balancer-controller/values.yaml

```diff
@@ -326,6 +326,10 @@ serviceMonitor:
   # Namespace to create the service monitor in
   namespace:
 
+#prometheusScrapping enable Promehteus Scrapping configuration in annotation section
```

Comment:
This does not adequately describe the end-effect of the setting. It should describe the behavior, not the mechanism.